neqo-http3/src/server.rs

```diff
@@ -138,6 +138,7 @@ impl Http3Server {
         max_datagrams: NonZeroUsize,
     ) -> OutputBatch {
         qtrace!("[{self}] Process");
+        self.process_http3(now);
```

Comment:
Force the **server** to flush down the WebTransport Session 200 OK to the QUIC layer. Without it the server will first send the WebTransport datagram and then the WebTransport Session OK.

Will find a fix before merging here.

The proof of the test is still valid, i.e. the client does handle a WebTransport datagram when it is packed along with the Webtransport session 200 OK in the same QUIC packet and UDP datagram.